common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -182,6 +185,55 @@ def test_add_entitlement_and_upgrade_audit_enrollment(self, mock_get_course_runs
         assert course_entitlement.enrollment_course_run == enrollment
         assert results == CourseEntitlementSerializer(course_entitlement).data
 
+    @patch("entitlements.api.v1.views.get_course_runs_for_course")
+    def test_add_entitlement_and_upgrade_audit_enrollment_with_dynamic_deadline(self, mock_get_course_runs):
+        """
+        Verify that if an entitlement is added for a user, if the user has one upgradeable enrollment
+        that enrollment is upgraded to the mode of the entitlement and linked to the entitlement regardless of
+        dynamic upgrade deadline being set.
+        """
+        DynamicUpgradeDeadlineConfiguration.objects.create(enabled=True)
+        course = CourseFactory.create(self_paced=True)
+        course_uuid = uuid.uuid4()
+        course_mode = CourseModeFactory(
+            course_id=course.id,
+            mode_slug=CourseMode.VERIFIED,
+            # This must be in the future to ensure it is returned by downstream code.
+            expiration_datetime=now() + timedelta(days=1)
```

Comment:
hypernit: since there are no spaces around the assignment operator, there should also not be any spaces around the additive operator.

Reply:
@McKenzieW This breaks the pep8 quality standards.  According to pep8 you need a space for the arithmetic operation and no space for setting `expiration_datetime=` in the method call.

Reply:
That's annoying but I understand